package/gluon-web-admin/i18n/de.po

```diff
@@ -17,7 +17,7 @@ msgid "Advanced settings"
 msgstr "Erweiterte Einstellungen"
 
 msgid ""
-"Alternatively, you can set a password to access you node. Please choose a "
+"Alternatively, you can set a password to access your node. Please choose a "
```

Comment:
This is an unrelated change that affects multiple files. Please split it out into a second commit.